build.sh

```diff
@@ -7,5 +7,5 @@ sed -i -e "s%#syslogAddress#%`echo $SYSLOG_HOST`%g" ${pwd}/conf.json
 sed -i -e "s%#apisixBaseUrl#%`echo $APISIX_BASE_URL`%g" ${pwd}/conf.json
 
 cd /root/ingress-controller
-exec ./ingress-controller
+exec ./ingress-controller ingress
```

Comment:
why we need the argument `ingress`?

Reply:
We put all core logic about "ingress controller" in the sub command ingress, and we can extend this tool by adding others sub command, for instance, some debugging tools to fetch its internal states, or adding the service apis support by the "gateway" sub command. So using sub commands is more flexible.